### DISCUSSION TITLE
Multi tasking: how many tabs are open

### DISCUSSION THREAD

**Discussion by dhrumil83** [microsoft/Microsoft-Fabric-workload-development-sample#174] [Q&A] [answered]
Is there an API, that can tell us how many tabs are currently open?

We have a "Save as" option that can create a new item/copy from an existing item, but if the maximum number of tabs is already open, we need a way to warn users to close a tab before they save an item else they cannot navigate to it as part of save as.

**Comment by zainadr** [comment, marked as answer]
We do not have an API that can tell how many opened tabs, but once you set the maxInstanceCount  to "n", and try to open the "n+1" item a message will pop up telling you cannot open more than "n" items" at the same time.
